.github/workflows/ci.yml

```diff
@@ -14,6 +14,16 @@ env:
   TEST_TIMEOUT: 360
 
 jobs:
+  set_env:
```

Comment:
Adding a separate job for just one variable looks redundant to me

Reply:
Updated